front/src/routes/dashboard/edit-dashboard/EditBoxColumns.jsx

```diff
@@ -8,145 +8,187 @@ import EditBox from './EditBox';
-            'btn-warning': props.isMobileReordering
-          })}
-          onClick={props.toggleMobileReorder}
-        >
-          <i class="fe fe-list mr-2" />
-          {!props.isMobileReordering && <Text id="dashboard.reorderDashboardButton" />}
-          {props.isMobileReordering && <Text id="dashboard.stopReorderingDashboardButton" />}
-        </button>
+      <div class="row mb-4">
+        <div class="col-md-4 d-lg-none d-xl-none">
+          <button
+            class={cx('btn', {
+              'btn-secondary': !props.isMobileReordering,
+              'btn-warning': props.isMobileReordering
+            })}
+            onClick={props.toggleMobileReorder}
+          >
+            <i class="fe fe-list mr-2" />
+            {!props.isMobileReordering && <Text id="dashboard.reorderDashboardButton" />}
+            {props.isMobileReordering && <Text id="dashboard.stopReorderingDashboardButton" />}
+          </button>
+        </div>
       </div>
-    </div>
-    <DndProvider backend={props.isTouchDevice ? TouchBackend : HTML5Backend}>
-      {props.isMobileReordering && <AutoScrollMobile position="top" box_type={DASHBOARD_EDIT_BOX_TYPE} />}
-      <div class="d-flex flex-row flex-wrap justify-content-center">
-        {props.homeDashboard &&
-          props.homeDashboard.boxes &&
-          props.homeDashboard.boxes.map((column, x) => (
-            <div
-              class={cx('d-flex flex-column col-lg-4', style.removePadding, {
-                [style.removePaddingFirstCol]: x === 0,
-                [style.removePaddingLastCol]: x === 2
-              })}
-            >
-              <h3 class="text-center">
-                <Text id="dashboard.boxes.column" fields={{ index: x + 1 }} />
-              </h3>
-
-              {column.length > 0 && (
-                <div>
-                  {column.map((box, y) => (
-                    <EditBox {...props} box={box} x={x} y={y} isMobileReordering={props.isMobileReordering} />
-                  ))}
-                  <BottomDropZone
-                    moveCard={props.moveCard}
-                    x={x}
-                    y={column.length}
-                    isMobileReordering={props.isMobileReordering}
-                  />
+      <DndProvider backend={props.isTouchDevice ? TouchBackend : HTML5Backend}>
+        {props.isMobileReordering && <AutoScrollMobile position="top" box_type={DASHBOARD_EDIT_BOX_TYPE} />}
+        <div class={cx('d-flex flex-row align-items-start', style.columnsCard)}>
+          {props.homeDashboard &&
+            props.homeDashboard.boxes &&
+            props.homeDashboard.boxes.map((column, x) => (
+              <div
+                class={cx('d-flex flex-column', style.column, stylePrimary.removePadding, {
+                  [stylePrimary.removePaddingFirstCol]: x === 0,
+                  [stylePrimary.removePaddingLastCol]: x === maxBoxes,
+                  [style.removePadding]: true // it will remove padding when in mobile view
```

Comment:
No need for conditional if it's always true, you can put that in the top : 

```
cx('d-flex flex-column', style.column, stylePrimary.removePadding, style.removePadding, ...
```

Reply:
But I can't get rid of this problem related to `overflow: hidden;` :
![image](https://github.com/user-attachments/assets/7aa312e1-2fe5-44a4-aa81-054e4ae8d73a)

Reply:
> But I can't get rid of this problem related to `overflow: hidden;` : ![image](https://private-user-images.githubusercontent.com/35010958/363036701-7aa312e1-2fe5-44a4-aa81-054e4ae8d73a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ5OTE1NTgsIm5iZiI6MTcyNDk5MTI1OCwicGF0aCI6Ii8zNTAxMDk1OC8zNjMwMzY3MDEtN2FhMzEyZTEtMmZlNS00NGE0LWFhODEtMDU0ZTRhZThkNzNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODMwVDA0MTQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQwNjExMWU3NjgyNGRjMDFkY2QxMDM5MjY3ZjNmMzE0NmI4MzgxNjgxNTNhNjc2YWEyZDNlOGUzOGE1NTk3ODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nDp_99gvCo-MfsDR3hUnQpT07hQaAYHKhwWjZ135Cjo)

Bon bah visiblement ça ne vaut que pour les boxes "Appareils" et "Appareils de la pièce" car par exemple pour la sélection : 
![image](https://github.com/user-attachments/assets/ba44d1c1-d65f-4e1e-aa0b-02b928cecd5d)
Pas de soucis avec les alarmes par exemple : 
![image](https://github.com/user-attachments/assets/154f1803-ac1a-48be-96a0-3639f711dbef)

Donc je suppute que c'est le dropdown `import Select from 'react-select';` qui me pose problème.